lib/trackler/problem.rb

```diff
@@ -10,78 +10,99 @@ def initialize(slug, root)
     end
 
     def exists?
-      !!md && !!yaml
+      !!description && !!metadata
```

Comment:
I think this makes much more sense now the methods have more meaningful names and it is checking for the existence of the actual data rather than the path to that data.